project-charter.md

```diff
@@ -30,13 +30,13 @@ Provide information on how the development and distribution of the Project Chart
 
 ### 1.1 Purpose of Project Charter
 
-The U↑ project charter documents and tracks the necessary information required by decision maker(s) to approve the project for funding. The project charter should include the needs, scope, justification, and resource commitment as well as the project’s sponsor(s) decision to proceed or not to proceed with the project. It is created during the Initiating Phase of the project.
+The U↑ project charter documents and tracks the necessary information required by the decision maker(s) to approve the project for funding. The project charter should include the needs, scope, justification, and resource commitment as well as the project’s sponsor(s) decision to proceed or not to proceed with the project. It is created during the Initiating Phase of the project.
 
 The intended audience of the U↑ project charter is the project sponsor and senior leadership.
 
 ## 2 PROJECT AND PRODUCT OVERVIEW
 
-U↑ is an internet-based phone system for company employees. Employees can make and receive calls over the internet using their desktop computers, smartphones, tablets, or other devices connected to the U↑ system. Each user's devices connected to U↑ will be associated with a unique business phone number for incoming and outgoing calls. Any connected device can make or receive calls using U↑ whenever and wherever necessary.
+U↑ is an internet-based phone system for company employees. Employees can make and receive calls over the Internet using their desktop computers, smartphones, tablets, or other devices connected to the U↑ system. Each user's device connected to U↑ will be associated with a unique business phone number for incoming and outgoing calls. Any connected device can make or receive calls using U↑ whenever and wherever necessary.
```

Comment:
not 100% sure this needs to be fixed, I was under the impression a single user can connect multiple devices (e.g. tablet, phone, laptop, etc.) and as long as they belong to the same person they'll have the same phone number. curious how others interpreted it

Reply:
Ah my bad, I just changed it without giving it much thought after Grammarly flagged the wording.

Would this be better?
```
Each of the user's devices connected to...
```